src/crypto/crypto-ops.c

```diff
@@ -3831,14 +3831,39 @@ int sc_isnonzero(const unsigned char *s) {
 }
 
 int ge_p3_is_point_at_infinity(const ge_p3 *p) {
```

Comment:
Is this a good opportunity to relabel this with `_vartime` at the end? This would follow the typical naming scheme for crypto functions.

Reply:
Sounds good to me!